dockerfiles/theia/Dockerfile

```diff
@@ -18,7 +18,7 @@ WORKDIR ${HOME}
 ARG GITHUB_TOKEN=''
 ENV GITHUB_TOKEN=$GITHUB_TOKEN
 
-ARG THEIA_GITHUB_REPO=eclipse-theia/theia
+ARG THEIA_GITHUB_REPO=
```

Comment:
what happens if we call build.sh from dockerfiles/theia folder ?

Reply:
got it :slightly_smiling_face: - updated
Thanks!